pento/free-as-in-speech
# Wix: Static Pages

This adds preliminary static page parsing.

### Testing instructions
- Build the extension with `npm run build` and load it into your browser.
- Load your Wix dashboard
- Open the extension window and click export
- The WXR should contain your static pages.
